src/docs/customize/translations.md

```diff
@@ -1,9 +1,3 @@
----
-name: Translations
-menu: Customize
-route: /customize/translations
----
-
 # Translations
```

Comment:
It is out if scope, but can you please update structure example on this page according to translations?

Reply:
fixed